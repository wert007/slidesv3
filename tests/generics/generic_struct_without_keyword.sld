// This should be either a compile error or work. right now it does neither.
struct A {
    value: $Type;
}

func main() {
    let a = new A(3);
}