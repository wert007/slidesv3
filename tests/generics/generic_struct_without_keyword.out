Error in ..\tests\generics\generic_struct_without_keyword.sld:3:12: Used generic type in struct A without declaring it as 'generic'.
Error in ..\tests\generics\generic_struct_without_keyword.sld:7:19: Cannot convert type int literal to type $Type.
